core/audio/start.sh

```diff
@@ -86,4 +86,9 @@ if [[ -n "$SOUND_ENABLE_SOUNDCARD_INPUT" ]]; then
   route_input_source
 fi
 
-exec pulseaudio --file /etc/pulse/balena-sound.pa
\ No newline at end of file
+exec pulseaudio \
+  --realtime=true \
+  --system=true \
```

Comment:
As you saw in #277, using `--system` results in all sort of errors and overall weird behaviour. I spent some time trying to make it work but got mixed results. More so, PulseAudio devs strongly discourage using it.

I've documented the decision of not using the system flag here: https://github.com/balenablocks/audio/issues/8